### DISCUSSION TITLE
Merko theme error

### DISCUSSION THREAD

**Discussion by Sakhi29** [anuraghazra/github-readme-stats#3134] [Q&A] [answered]
### Describe the bug

There's this bug in merko theme for github stats. Here's the screenshot for it. 

### Expected behavior

It should show the github stats like stars earned, no of contributions, PRs etc.

### Screenshots / Live demo link

![image](https://github.com/anuraghazra/github-readme-stats/assets/106910881/96dda36a-986b-496a-9743-fc29fe7bb9fe)

**Comment by rickstaa** [comment, marked as answer]
Hey @Sakhi29. This is caused by the rate-limited public instance by GitHub (See https://github.com/anuraghazra/github-readme-stats/issues/1471#issue-1062727627). As explained in this issue, until the [GitHub action](https://github.com/anuraghazra/github-readme-stats/issues/2179) version is released, you can fix this by [deploying your own vercel instance](https://github.com/anuraghazra/github-readme-stats#deploy-on-your-own).

**Reply by Sakhi29** [reply]
How can I fix it?

**Reply by rickstaa** [reply]
Sorry for being unclear. I updated my answer 👍🏻.

**Reply by Sakhi29** [reply]
Okay, thanks.

**Reply by rickstaa** [reply]
Feel free to let me know if you run into any problems and accept my answer if it solves your issues.
